test/unit/dependency.test.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+
+import { dependencies } from '../../package.json';
+
+const EXPECTED_DEPENDENCIES = ['bson', 'denque', 'mongodb-connection-string-url', 'socks'];
```

Comment:
A couple of questions

- Do we want to check the versions too?  That might help us prevent accidental version bumps but it would also mean we could no longer rely on automated tools (like Snyk) to bump our dependencies for us.  I can go either way so I'm curious to hear your thoughts.
- The ticket mentions dev dependencies.  Did you choose not to check our dev dependencies as well?  I personally don't think it's necessary, I was just curious.

Reply:
1. I didn't check versions specifically for the Snyk reason.
2. I also didn't think doing dev dependencies was necessary.